packages/base-zone/README.md

```diff
@@ -0,0 +1,9 @@
+# Base Zone Library
```

Comment:
I was surprised it's not called endo-zone but I agree this is better. The plan to eventually move to Endo needn't be captured in SCM. There's sufficient motivation without it.